github-sync/github-data/users.yaml

```diff
@@ -466,6 +466,11 @@ users:
   - username: thelinuxfoundation
     role: admin
     teams: []
+  - username: tnytown
+    role: member
+    teams:
```

Comment:
I think `teams` should be `[]` and then update `repositories.yaml` to add him with triage rights under the specific repos. the `triage` team is a wider scope.

Reply:
Done! bf99f9e..b7868c8